locales/en.yml

```
@@ -43,18 +58,18 @@ en:
         description: Gay-friendly Airbnb. High growth website. Short iterations. Large feature set.
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
-        title: misterb&b
-      misterout:
+        title: Misterb&b
```

No, I think there's no uppercase: https://www.misterbandb.com/pages/join-us

```suggestion
        title: misterb&b
```